src/neo/IO/Caching/CloneCache.cs

```diff
@@ -24,10 +24,10 @@ protected override void DeleteInternal(TKey key)
             innerCache.Delete(key);
         }
 
-        protected override IEnumerable<(TKey, TValue)> FindInternal(byte[] key_prefix)
+        protected override IEnumerable<(TKey, TValue)> SeekInternal(byte[] key)
```

Comment:
Why rename this method?

Reply:
We want to use `FindRange(KeyStart, KeyEnd)` method, `FindInternal` dosen't support it.

Reply:
Maybe it is better to keep the previous name, no? I did not see any difference in the logic of the search.

Reply:
No, it's different. If we want to query [Prefix_50, Prefix_60) in range [Prefix_0, Prefix_100) and Prefix_50 dosen't exist, we can't seek the position at Prefix_51 by `Find`.

Reply:
But the logic is not changed.

Reply:
The logic of `Find` and `Seek`?

The implementation of `Find`
```cs
public static IEnumerable<T> Find<T>(this DB db, ReadOptions options, byte[] prefix, Func<byte[], byte[], T> resultSelector)
{
	using Iterator it = db.NewIterator(options);
	for (it.Seek(prefix); it.Valid(); it.Next())
	{
		byte[] key = it.Key();
		if (key.Length < prefix.Length) break;
		if (!key.AsSpan().StartsWith(prefix)) break;
		yield return resultSelector(key, it.Value());
	}
}
// https://github.com/neo-project/neo-modules/blob/d55d88c5b6eb3f44026237d73ea001c535def334/src/LevelDBStore/IO/Data/LevelDB/Helper.cs#L9-L19
```
The implementation of `Seek`
```cs
public static IEnumerable<T> Seek<T>(this DB db, ReadOptions options, byte[] prefix, Func<byte[], byte[], T> resultSelector)
{
	using Iterator it = db.NewIterator(options);
	for (it.Seek(prefix); it.Valid(); it.Next())
		yield return resultSelector(it.Key(), it.Value());
}
```

Reply:
Then you should not modify `Find()`. I think you'd better add a new method. And it is better to add a "direction" parameter.

You can create a new PR for it.

Reply:
Agree it could reduce the changes and make it easy the review

Reply:
But we don't need to support both `Find` and `Seek`, as `Find` can be derived from `Seek`, and I have reversed the `Find` method in DataCache. https://github.com/neo-project/neo/pull/1716/files#diff-81633e1e3a5fa6895bc87d2b5875f9a6R205